lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -860,6 +865,10 @@ def _create_deployment_parameters(
                 azure_node_runbook.purchase_plan = self._process_marketplace_image_plan(
                     azure_node_runbook.location, azure_node_runbook.marketplace
                 )
+
+            azure_node_runbook.enable_sriov = self._get_sriov_capability(
```

Comment:
It doesn't need to set it into runbook. It's confusing to put it in the schema, and not able to set it.